
<launch>
  <arg name="usb_port"     default="/dev/ttyUSB0"/>
  <arg name="baud_rate"    default="1000000"/>
  <arg name="control_period"         default="0.010"/>
  <arg name="use_platform"           default="true"/>
  <arg name="use_sim"           default="false"/>
  <arg name="with_gripper"           default="false"/>
  <arg name="arm_id"           default="1"/>

  <node name="open_manipulator_p_controller" pkg="open_manipulator_p_controller" type="open_manipulator_p_controller"
     output="screen" args="$(arg usb_port) $(arg baud_rate) $(arg arm_id)">
    <param name="using_platform"       value="$(arg use_platform)"/>
    <param name="using_sim"       value="$(arg use_sim)"/>
    <param name="control_period"       value="$(arg control_period)"/>
    <param name="with_gripper"         value="$(arg with_gripper)"/>
  </node>

</launch>